Dockerfile

```diff
@@ -50,6 +50,14 @@ COPY Taskfile.yml .
 # ========= Install cargo-tools =========
 RUN task install-cargo-tools
 
+# ========= Install hugo =========
+# https://stackoverflow.com/a/75330596/1827982 and
+# https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/
+
+ARG BUILDARCH
+RUN curl -L "https://github.com/gohugoio/hugo/releases/download/v0.110.0/hugo_0.110.0_linux-$BUILDARCH.deb" -o hugo.deb
+RUN apt-get install ./hugo.deb
```

Comment:
Some comments

- Do you need to fix the version? GitHub have URLs that specifies the latest version release so you can always install the latest version if you do that. (Hugo is released fairly frequently)
  - If you want to fix the version, it would be convenient to make the version a variable so that you do not have to edit two places.
- Since each use of the RUN clause adds a layer of images, it is sometimes better to perform the process within a single RUN clause, e.g. `RUN curl ... && apt-get install ...`
- Since Hugo is a stand-alone executable, there is also a way to download the executable instead of the deb package.

I apologize for not being able to offer specific suggestions for modifications, but I think the following script may be helpful.
https://github.com/devcontainers/features/blob/main/src/hugo/install.sh